apps/ledger-live-mobile/src/components/RootNavigator/AccountsNavigator.tsx

```diff
@@ -75,6 +79,24 @@ export default function AccountsNavigator() {
           headerShown: false,
         }}
       />
+      {accountListUIFF?.enabled && (
+        <>
+          <Stack.Screen
+            name={ScreenName.AccountsList}
+            component={AccountsList}
+            options={{
+              headerTitle: "",
+            }}
+          />
+          <Stack.Screen
+            name={ScreenName.AssetsList}
+            component={AssetsList}
+            options={{
+              headerTitle: "",
+            }}
+          />
+        </>
+      )}
       <Stack.Screen
```

Comment:
this screen will be there even when the FF is off  ?

Reply:
Yes, it's reborn state here :)

Reply:
okay thanks